src/core/shape_inference/include/fft_common_validation.hpp

```diff
@@ -73,8 +72,8 @@ void validate_axes(const ov::op::util::FFTBase* op,
     // according to the RDFT operation specification, axes should be integers from -r to (r - 1)
     // inclusively, where r = rank(data). A negative axis 'a' is interpreted as an axis 'r + a'.
     const int64_t axis_correction = (fft_kind == FFTKind::RealInput) ? input_rank : (input_rank - 1);
-    auto axis_min_value = -static_cast<int64_t>(input_rank);
-    auto axis_max_value = static_cast<int64_t>(input_rank) - 1;
+    auto axis_min_value = -input_rank;
+    auto axis_max_value = input_rank - 1;
```

Comment:
```cpp
    ov::normalize_axes(op, axis_correction, axes);
    auto axes_set = AxisSet(std::vector<size_t>(axes.begin(), axes.end()));
    NODE_VALIDATION_CHECK(op, axes.size() == axes_set.size(), "Each axis must be unique.");
```
It looks like exceptions are thrown correctly in test, only message message is different.

Can you check if loop below can be replaced?

Reply:
Okay, looks legit, replaced.